src/assets/increasingSort.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns" viewBox="0 0 100 100" version="1.1" x="0px" y="0px"><title>Sort Ascending</title><desc>Created with Sketch.</desc><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" sketch:type="MSPage"><g sketch:type="MSArtboardGroup" fill="#000000"><g sketch:type="MSLayerGroup" transform="translate(8.000000, 4.000000)"><path d="M64,79 L56.7191654,87.0922045 L64.2221013,93.7032523 L78.7679977,77.1949874 L86.0708143,69 L75.0165311,69 L0,69 L0,79 L64,79 Z" sketch:type="MSShapeGroup"/><path d="M86,0 L76,0 L76,63 L86,63 L86,0 Z" sketch:type="MSShapeGroup"/><path d="M67,63 L67,13 L57,13 L57,63 L67,63 L67,63 Z" sketch:type="MSShapeGroup"/><path d="M48,63 L48,25 L38,25 L38,63 L48,63 L48,63 Z" sketch:type="MSShapeGroup"/><path d="M29,63 L29,38 L19,38 L19,63 L29,63 L29,63 Z" sketch:type="MSShapeGroup"/><path d="M10,63 L10,50 L0,50 L0,63 L10,63 L10,63 Z" sketch:type="MSShapeGroup"/></g></g></g>
+<!--<text x="0" y="115" fill="#000000" font-size="5px" font-weight="bold" font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif">Created by Chris</text><text x="0" y="120" fill="#000000" font-size="5px" font-weight="bold" font-family="'Helvetica Neue', Helvetica, Arial-Unicode, Arial, Sans-serif">from the Noun Project</text>-->
+<!-- changed the height in viewBox to 125 to 100 -->
+</svg>
```

Comment:
These two icons are pretty difficult to identify in the `HoverField`. I've had more luck with the [FontAwesome icon set](http://fontawesome.io/icons/). You can find SVGs for them [here](https://github.com/encharm/Font-Awesome-SVG-PNG/tree/master/black/svg). 

With regards to _which_ icon to show (and I'm now referring to the corresponding FontAwesome icons here), let's start by showing `sort` (double arrow) to indicate that one _can_ sort a field, but that it has _not yet_ been sorted. Once a user clicks the icon, the icon should then switch to indicate the current sort order (i.e., should show ascending if ascending and descending if descending). Let's also use the icon most appropriate for the field type (i.e., `sort-alpha-` for categorical fields, and `sort-numeric-` for the other two).